CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.9.0 - 2021-11-11
+
+### Added
+- Added dynamic debug configurations for debugging a static web app using the Azure Static Web Apps CLI [#557](https://github.com/microsoft/vscode-azurestaticwebapps/pull/557)
+
+    ![Authorize Access to through GitHub](resources/changelog/dynamic_debug_configs.png)
+
+- Added autocomplete and validation for [swa-cli.config.json](https://github.com/Azure/static-web-apps-cli#swa-cliconfigjson-file) files
```

Comment:
I think you can omit "Added" here and on line 6 above since it's already in the "Added" section